pandas/core/frame.py

```diff
@@ -4479,6 +4479,14 @@ def quantile(self, q=0.5, axis=0, numeric_only=True):
             0 <= q <= 1, the quantile(s) to compute
         axis : {0, 1}
             0 for row-wise, 1 for column-wise
+        interpolation : {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
+            Specifies the interpolation method to use, when the desired quantile lies between two data points i and j:
```

Comment:
Can you keep here to PEP8 line length?

Further, better to make a bullet points list of the options (newlines are ignored when rendered to html)